ui/components/multichain/network-manager/components/edit-network/EditNetwork.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useCallback } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+import {
+  ButtonIcon,
+  ButtonIconSize,
+  IconName,
+  Text,
+  Box,
+} from '../../../../component-library';
+import {
+  AlignItems,
+  Display,
+  FlexDirection,
+  JustifyContent,
+  TextVariant,
+} from '../../../../../helpers/constants/design-system';
+
+export const EditNetwork: React.FC = () => {
+  const history = useHistory();
+  const { id } = useParams<{ id: string }>();
+
+  const handleGoBack = () => {
+    history.push('/');
+  };
+
+  const handleSaveNetwork = useCallback(() => {
+    console.log('save network with id:', id);
+    // TODO: Implement save logic
+    // Navigate back to main network manager after save
+    handleGoBack();
+  }, [handleGoBack, id]);
+
+  const handleDeleteNetwork = useCallback(() => {
+    console.log('delete network with id:', id);
```

Comment:
remove logs (I think this is all I could see)

Reply:
This is also a bit that I didn't author, but I don't think that this function is needed at all. I was noticing some wonky behavior on network removal though.

On the other hand, it's also a bit wonky on main, so maybe we should just fix it.